dist/angular-localization.js

```diff
@@ -21,8 +21,7 @@ angular.module('ngLocalize.InstalledLanguages', [])
     });
 angular.module('ngLocalize')
     .service('locale', ['$injector', '$http', '$q', '$log', '$rootScope', '$window', 'localeConf', 'localeEvents', 'localeSupported', 'localeFallbacks', function ($injector, $http, $q, $log, $rootScope, $window, localeConf, localeEvents, localeSupported, localeFallbacks) {
-        var TOKEN_REGEX = new RegExp('^[\\w\\.-]+\\.[\\w\\s\\.-]+\\w(:.*)?$'),
-
+        var TOKEN_REGEX = (localeConf.validTokens | new RegExp('^[\\w\\.-]+\\.[\\w\\s\\.-]+\\w(:.*)?$')),
```

Comment:
If you're doing OR you need `||` your way is a bitwise compare. Also, in this case it's set so no need to explicitly write it in here as well. Simly

```js
var TOKEN_REGEX = localeConf.validTokens;
```